core/src/main/java/com/linecorp/armeria/common/multipart/MultipartDecoder.java

```diff
@@ -69,8 +90,9 @@ public void processOnError(Throwable cause) {
         if (parser != null) {
             try {
                 parser.close();
-            } catch (MimeParsingException ex) {
-                logger.warn(ex.getMessage(), ex);
+            } catch (MimeParsingException ignore) {
+                // There is a cause already happened and passed to subscriber already
+                // So the parsing exception is not important for user and miss leading if we log here.
```

Comment:
How about rephrasing like this:

```suggestion
                // This exception has been passed to the subscriber already and will be handled there,
                // so we shouldn't log it because otherwise it may mislead a user.
```